oddrn_generator/path_models.py

```diff
@@ -70,51 +73,71 @@ class Config:
             "views_columns": ("databases", "schemas", "views", "views_columns"),
             "relationships": ("databases", "schemas", "tables", "relationships"),
         }
-        data_source_path = "databases"
+
+    data_source_path: str = "databases"
+    dependencies_map: dict = Field(
+        default_factory=lambda: PostgresqlPathsModel._dependencies_map_factory()
+    )
 
 
 class MysqlPathsModel(BasePathsModel):
     databases: str
-    tables: Optional[str]
-    views: Optional[str]
-    tables_columns: Optional[str] = Field(alias="columns")
-    views_columns: Optional[str] = Field(alias="columns")
-
-    class Config:
-        dependencies_map = {
+    tables: Optional[str] = None
+    views: Optional[str] = None
+    tables_columns: Optional[str] = Field(None, alias="columns")
+    views_columns: Optional[str] = Field(None, alias="columns")
+
+    @classmethod
+    def _dependencies_map_factory(cls):
+        return {
             "databases": ("databases",),
             "tables": ("databases", "tables"),
             "views": ("databases", "views"),
             "tables_columns": ("databases", "tables", "tables_columns"),
             "views_columns": ("databases", "views", "views_columns"),
         }
-        data_source_path = "databases"
+
+    data_source_path: str = "databases"
+    dependencies_map: dict = Field(
+        default_factory=lambda: MysqlPathsModel._dependencies_map_factory()
+    )
```

Comment:
```suggestion
    dependencies_map: dict = Field(
        default_factory=MysqlPathsModel._dependencies_map_factory
    )
```

Reply:
just in case If you would like elaborate type hint usage in python you can do something like this
```suggestion
DependenciesMap = Dict[str, Tuple[str, ...]]
    ...
    dependencies_map: DependenciesMap = Field(
        default_factory=lambda: MysqlPathsModel._dependencies_map_factory
    )
```